library/src/test/java/com/novoda/downloadmanager/LiteBatchFileBuilderTest.java

```diff
@@ -0,0 +1,99 @@
+package com.novoda.downloadmanager;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class LiteBatchFileBuilderTest {
+
+    private static final String ANY_NETWORK_ADDRESS = "http://ak.assets.com/some-file";
+
+    private final InternalBatchBuilder batchBuilder = mock(InternalBatchBuilder.class);
+    private final StorageRoot storageRoot = mock(StorageRoot.class);
+    private final DownloadBatchId downloadBatchId = mock(DownloadBatchId.class);
+
+    private LiteBatchFileBuilder liteBatchFileBuilder;
+
+    @Before
+    public void setUp() {
+        when(downloadBatchId.rawId()).thenReturn("my-movie");
+        liteBatchFileBuilder = new LiteBatchFileBuilder(
```

Comment:
Could move this up to the field and make it `final`.

Reply:
Or move it locally with a `given` factory method which takes in the conditions of the subject

Reply:
Done in 469aca79cb5739c28aa31d4338a2022f46d89ceb.